### PR TITLE
Allow rowId 'sum' with jr_get_subtable_value

### PR DESCRIPTION
**Pull request by nemchik**
Documented in 5.1 jr_get_subtable_value https://portal.jobrouter.com/documentation/en/5.1/javascript_api/index.html?jr_get_subtable_value.htm
Also works with jr_set_subtable_value although not documented

### CODE REVIEW

File: api.d.ts [hunk 1]
```diff
@@ -38,9 +38,9 @@ declare const jr_is_readonly: (id: ElementId, customHandler?: CustomErrorHandler
 declare const jr_set_disabled: (id: ElementId, value?: boolean, customHandler?: CustomErrorHandler) => void;
 declare const jr_is_disabled: (id: ElementId, customHandler?: CustomErrorHandler) => boolean;
 declare const jr_set_background_color: (id: ElementId, color: string, customHandler?: CustomErrorHandler) => void;
-declare const jr_set_subtable_value: (subtableViewName: string, rowId: number, columnName: string, value: any, customHandler?: CustomErrorHandler) => void;
+declare const jr_set_subtable_value: (subtableViewName: string, rowId: number | "sum", columnName: string, value: any, customHandler?: CustomErrorHandler) => void;
```
Comment (jmerchel) [inline api.d.ts]: This "feature" is not documented as we do not want users to set the sum row manually.
To update the sum row, we recommend using jr_sum_subtable_column as it does the calculation for you.

File: api.d.ts [hunk 2]
```diff
@@ -38,9 +38,9 @@ declare const jr_is_readonly: (id: ElementId, customHandler?: CustomErrorHandler
...
 declare const jr_set_subtable_display_value: (subtableViewName: string, rowId: number, columnName: string, value: any, customHandler?: CustomErrorHandler) => void;
-declare const jr_get_subtable_value: (subtableViewName: string, rowId: number, columnName: string, customHandler?: CustomErrorHandler) => any;
+declare const jr_get_subtable_value: (subtableViewName: string, rowId: number | "sum", columnName: string, customHandler?: CustomErrorHandler) => any;
```
Comment (jmerchel) [inline api.d.ts:43]: Good catch. This should be part of the api.

**Review by jmerchel — CHANGES_REQUESTED** [review]
Please revert the changes to jr_set_subtable_value. For explanation see my comment.

File: api.d.ts (hunk 1 above)
Comment (nemchik) [reply api.d.ts]: Resolved in https://github.com/jobrouter/javascript-api/pull/3/commits/6bbfda3aa249cb15377687df87469f2fd82897cd

**Comment by nemchik** [comment]
Ready for second review.

**Review by jmerchel — APPROVED** [review]

**Pull request merged**